tests/braille-specs/uz.yaml

```diff
@@ -25,21 +25,27 @@ tests:
   - # Sample for testing "ҳ", "қ" and "ў"
     - ҳуқуқларда
     - ⠹⠥⠽⠥⠽⠇⠁⠗⠙⠁
-  - # Sample for testing "sh"
+  - # Sample for testing "sh" inside a word/between other letters.
     - qilishlari
-    - ⠽⠊⠇⠊⠎⠹⠇⠁⠗⠊
-  - # Sample for testing "ch"
+    - ⠽⠊⠇⠊⠱⠇⠁⠗⠊
+  - # Sample for testing "ch" inside a word/between other letters.
     - anchadan
-    - ⠁⠝⠉⠹⠁⠙⠁⠝
+    - ⠁⠝⠟⠁⠙⠁⠝
   - # Phrase one
     - Hayirli kech
-    - ⠹⠁⠯⠊⠗⠇⠊⠀⠅⠑⠉⠹
+    - ⠹⠁⠯⠊⠗⠇⠊⠀⠅⠑⠟
   - # Phrase two
     - Рождество байрамингиз қутлуғ бўлсин
     - ⠗⠕⠚⠙⠑⠎⠞⠺⠕⠀⠃⠁⠯⠗⠁⠍⠊⠝⠛⠊⠵⠀⠽⠥⠞⠇⠥⠻⠀⠃⠧⠇⠎⠊⠝
-  - # Test for o’ character
-    - o’
-    - ⠧
-  - # Test for g‘ character
-    - g‘
-    - ⠻
+  - # Test for o’ character (with different accent marks)
+    - o’o`o‘
+    - ⠧⠧⠧
+  - # Test for g‘ character (with different accent marks)
+    - g‘g`g’
+    - ⠻⠻⠻
+  - # Corrected symbols
```

Comment:
I would just say something like "Various symbols". That they have been corrected isn't really relevant for the test file.